common/constants.js

```diff
@@ -3,12 +3,16 @@ const TYPE_INTERNAL_USER = 'UserRecord'
 const TYPE_VET_PRACTICE_RECORD = 'VetPracticeRecord'
 const TYPE_VET_ROLE = 'VetRole'
 const TYPE_VET_PRIMARY_ADMIN_ROLE = 'VetPrimaryAdminRole'
+const TYPE_USER_ADMIN_ROLE = 'UserAdminRole'
+const TYPE_USER_PRIMARY_ADMIN_ROLE = 'UserPrimaryAdminRole'
+const TYPE_USER_AUTHORISED_ROLE = 'UserAuthorisedRole'
```

Comment:
In the service we have
public enum Roles
    {
        PrimaryAdmin,
        Admin,
        AuthorisedUser
    }
Sounds silly but to stick to your naming pattern it is UserAuthorisedUserRole